components/Chat/ClientChatPage.tsx

```
@@ -8,12 +8,28 @@ import { ModuleWithResources } from "@/lib/actions";
+const ChatMessages = dynamic(() => import("./ChatMessages"), {
+  loading: () => (
+    <div className="flex-1 overflow-y-auto p-4">
+      <Loader2 className="h-8 w-8 animate-spin mx-auto" />
+    </div>
+  ),
+  ssr: false,
+});
+
+const WelcomeScreen = dynamic(() => import("./WelcomeScreen"), {
+  loading: () => (
+    <div className="text-center flex items-center justify-center h-96"></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)